site/src/test/java/uk/nhs/digital/test/util/RandomHelper.java

```diff
@@ -0,0 +1,40 @@
+package uk.nhs.digital.test.util;
+
+import org.apache.commons.lang3.ArrayUtils;
+
+import java.util.List;
+import java.util.Random;
+import java.util.UUID;
+import java.util.stream.IntStream;
+
+import static java.util.Collections.shuffle;
+import static java.util.stream.Collectors.toList;
+
+public class RandomHelper {
```

Comment:
Have a similar RandomHelper file in acceptance-test module.  CMS module could do with something similar for all the newRandomStrings. Could perhaps have 1 place for all these at some point going forward although not a priority for now though.

Reply:
I totally, agree, @ngta1.

I'll be raising a JIRA ticket to sort this out. I'm thinking of some shared 'test utils' or 'test resources' module.